### PR TITLE
Export lyrics

### PR DESCRIPTION
**Pull request by jtmobs**

### CODE REVIEW

**Comment by HaiDang666** [comment]
you install package in wrong directory, It should be in `packages/app/package.json`

**Comment by jtmobs** [comment]
Done, Please check now

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuclear",
-  "version": "0.6.13",
```
Comment (HaiDang666) [inline package.json:3]: You have to check closely the changes on `package.json`. It changes many packages to lower version. It should only add the new packages that u want to use.

**Review by HaiDang666 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jtmobs) [reply package.json:3]: Yes it changed mine to "version": "0.6.11",
should I commit the .11 version or leave it as it is?
jspdf library is the needed package and its there, let me know what exactly you want me to do

Comment (nukeop) [reply package.json:3]: I don't get it, why did you lower the version number? This should stay unchanged

Comment (jtmobs) [reply package.json:3]: I had issues with the forked one and we forked that months ago, I made adjustments and everything is fine now, please review

Comment (jtmobs) [reply package.json:3]: I had issues with the forked one and we forked that months ago, I made adjustments and everything is fine now, please review

Comment (nukeop) [reply package.json:3]: It looks like those changes are still there, can you merge master into your branch?

**Pull request closed without merging**